cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -336,15 +342,67 @@ pub type LocalAndForeignAssets = fungibles::UnionOf<
 	AccountId,
 >;
 
-/// Union fungibles implementation for [`LocalAndForeignAssets`] and `Balances`.
-pub type NativeAndAssets = fungible::UnionOf<
+/// Union fungibles implementation for `AssetsFreezer` and `ForeignAssetsFreezer`.
+pub type LocalAndForeignAssetsFreezer = fungibles::UnionOf<
+	AssetsFreezer,
+	ForeignAssetsFreezer,
+	LocalFromLeft<
+		AssetIdForTrustBackedAssetsConvert<TrustBackedAssetsPalletLocation, xcm::v4::Location>,
+		AssetIdForTrustBackedAssets,
+		xcm::v4::Location,
+	>,
+	xcm::v4::Location,
```

Comment:
`xcm::v5::Location` everywhere, but when you rebase/merge master, it should be fixed